scapy/layers/dns.py

```diff
@@ -32,7 +35,6 @@ def i2m(self, pkt, x):
 
         # Truncate chunks that cannot be encoded (more than 63 bytes..)
         x = "".join(chr(len(y)) + y for y in (k[:63] for k in x.split(".")))
```

Comment:
I am concerned about this as chr will return a unicode string in python3. But maybe this is not addressed in this PR?